_pages/plugins/focust.md

```diff
@@ -0,0 +1,21 @@
+---
+title: FOCUST
+name: "FOCUST"
+description: Fluorescent Object and Cell gpU-accelerated Segmentation Toolbox.
+categories: [Analysis,Segmentation]
+website: "https://sebastianamos.github.io/FOCUST-Plugin-Site/"
+update-site: "FOCUST"
+team-founder: '@SebastianAmos'
+team-maintainer: '@SebastianAmos'
+source-url: https://github.com/SebastianAmos/FOCUST-Plugin
```

Comment:
@SebastianAmos this link shows a 404 error for me. Is the repository still private? If so, can you please make it public?

Reply:
@imagejan - whoops! Should be sorted now.
Cheers
Seb